spec/json_matchers/match_response_schema_spec.rb

```diff
@@ -24,43 +24,43 @@
 
   it "accepts options for the validator" do
     create_schema("foo_schema", {
-      "type" => "object",
-      "required" => [
+      "type": "object",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.